scratchpad2.R

```diff
@@ -0,0 +1,4 @@
+library(repgen)
```

Comment:
Yes to both, they are helpful for reference when not building with RStudio. I can move them if there is a better place for them, but they were supposed to just be helper files during development. I could be convinced to drop them also if @lindsaycarr or @jiwalker-usgs don't like them either.